### DISCUSSION TITLE
Roman aqueducts

### DISCUSSION THREAD

**Discussion by docalb** [BenWestgate/Bails#49] [General]
If we are successful in bringing about a bitcoin standard, we will have created the tooling (bails, etc) that will be used in one form or another for, what, the next 1000 plus years? 

Imagine having designed/built Roman aqueducts, providing value for millennia, long after those builders passed.

Amazing moment in time.

**Comment by BenWestgate** [comment]
Yea, I do see projects like Bails as playing a pivotal role in securing the bitcoin standard against attacks by agents with entrenched interests against it becoming global money.

These other wallets with no full validation, no transaction privacy, no Tor, no plausible deniability just won't work if attacks against users escalate to the level that they could.

By having an escape valve, we make sure such attacks are never tried at scale because they will be unprofitable due to easy-to-use  defensive tools.
